### ISSUE TITLE
Build issue with google play services (possible conflict with cordova-plugin-google-analytics)

### ISSUE THREAD

**Issue by p-grzelczak** [intercom/intercom-cordova#31]
I'm having an issue with running cordova-plugin-intercom simultaneously with [cordova-plugin-google-analytics](https://github.com/danwilson/google-analytics-plugin) on Android. 

If I remove one of them, the other works just fine, but I can't get it to work (or actually build an android app) when both plugins are installed.

Here's the error from build process (either using `ionic run android` or `ionic build --release android`, or cordova equivalent):
```
:compileArmv7ReleaseJava
/Users/piotr_mac/Workspace/[appname]/platforms/android/src/io/intercom/android/sdk/IntercomGCMManager.java:4: error: cannot find symbol
import com.google.android.gms.common.GooglePlayServicesUtil;
                                    ^
  symbol:   class GooglePlayServicesUtil
  location: package com.google.android.gms.common
/Users/piotr_mac/Workspace/[appname]/platforms/android/src/io/intercom/android/sdk/IntercomGCMManager.java:52: error: cannot find symbol
        return GooglePlayServicesUtil.isGooglePlayServicesAvailable(context) == ConnectionResult.SUCCESS;
               ^
  symbol:   variable GooglePlayServicesUtil
  location: class IntercomGCMManager
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors
 FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileArmv7ReleaseJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```
And I'm quite clueless on what should I do with that, google's not really helpful too.

I already have installed these:
Android Support Repository
Android Support Library
Google Play services
Google repository

Using:
Cordova 5.4.1

Min. Android SDK version: 19

Would really appreciate some help.

**Comment by alexanderwallin** [comment]
I had the same exact problem. I solved it by changing from a specific gcm version to a `+` (see the PR above).

**Comment by alexanderwallin** [comment]
@p-grzelczak Would you care to try `cordova plugin add https://github.com/alexanderwallin/intercom-cordova.git#682b34` and see if it works?

**Comment by p-grzelczak** [comment]
@alexanderwallin Hi, nice one - it went straight through and built an app without a problem.
Will use that for now until it gets merged, thanks a lot!

**Comment by jtreanor** [comment]
#32 has now been merged thanks to @alexanderwallin.

I have bumped the version and released an update on npm (version 1.1.4) :smile:
